### DISCUSSION TITLE
Collection Combining and Sharing: 2 parts

### DISCUSSION THREAD

**Discussion by DaydreamHero** [meisnate12/Plex-Meta-Manager#1691] [Q&A] [answered]
1. Can I combine 2 imdb_lists into one collection? For instance, if I (for some reason) wanted ALL DC IP in one Plex collection and I didn't want to create/maintain the lists, is there a way to take 
DCAMU's imdb_list: https://www.imdb.com/list/ls026910571/ and the DCEU's imdb_list: https://www.imdb.com/list/ls077827588/ and put them into one call like below (but actually working):
```
  DC All:
    imdb_list: https://www.imdb.com/list/ls026910571/, https://www.imdb.com/list/ls077827588/
    collection_order: release
    sync_mode: sync
    summary: "All DC IP in one place."
```

2. I may need to throw this somewhere else, but I'm new here and don't want to suggest something that already exists: Do we have a list sharing area? I know I can Google, using the above examples, "IMDB List DCEU" and get a series of lists, but I don't know if they are maintained (I also don't know if I'll agree with their logic, but that is secondary since that will be pervasive no matter what). I am curious if this community has a "hey I made this list that I'll continue to maintain" section where we can put our trakt, imdb, mdb, tmdb, etc. lists/collections for others to use.

**Comment by YozoraXCII** [comment, marked as answer]
For the first one, you would do it like this:

```
  DC All:
    imdb_list:
      - https://www.imdb.com/list/ls026910571/
      - https://www.imdb.com/list/ls077827588/
    collection_order: release
    sync_mode: sync
    summary: "All DC IP in one place."
```

For the second - not really, people often share their configs in our Discord server which can contain things like lists. We also have the Default Files which automates the creation most of the "big hitters" too.

EDIT: There is also the community Configs Repo:
https://github.com/meisnate12/Plex-Meta-Manager-Configs

**Reply by DaydreamHero** [reply]
1. Annoyingly, I had seen the hyphenates before and it didn't dawn on me that that was what I was looking for, thanks.
2. I'll have to check out the discord, I'd seen the configs, but it doesn't tell me that the lists are maintained (I found a few lists when I was searching that were great up until X and that will drive me crazy when I notice I'm no longer syncing.). Appreciate it.
